docs/tools/code-extension.mdx

```diff
@@ -131,7 +131,7 @@ Once the packages are installed, they will appear in the `./build/lfc_include/`
 
 In this structure:
 - **\<Package Name\>**: Represents the root folder of the main package.
-- **installed_package**: Represents each package listed in `Lingo.toml`, which contains one or more LF projects featuring reusable reactors.
+- **<Installed Package>**: Represents each package listed in `Lingo.toml`, which contains one or more LF projects featuring reusable reactors.
```

Comment:
Sorry the `<` and `>` need to be escaped. My bad!

Reply:
Yes, I’ve already corrected this part. I’ll make the modifications and commit all the updates together